### ISSUE TITLE
Slugs validated always as wrong

### ISSUE THREAD

**Issue by rokondo** [django-wiki/django-wiki#1030]
I am new to django wiki and I am not sure whether it is a bug or my mistake. The slug which is suggested always failed, for example: "abc-232-33" or "wechsel_2010_2020". The wiki is located in the path "/wiki/". I am using the latest version 0.6b1.

![27-02-_2020_15-40-55](https://user-images.githubusercontent.com/640866/75455108-a6a47f00-5977-11ea-850a-ffcfbb963a7f.png)

**Comment by benjaoming** [comment]
It sounds like your Project's `urls.py` is misconfigured.

What do you see when you navigate to the slug that you are typing in?

**Comment by benjaoming** [comment]
I am closing this, as it is not an issue. Please read https://github.com/django-wiki/django-wiki/blob/master/SUPPORT.rst

**Comment by rokondo** [comment]
> It sounds like your Project's `urls.py` is misconfigured.
> 
> What do you see when you navigate to the slug that you are typing in?

I configured at apps because I can not set url from the project:
`urlpatterns = [
  ....
    re_path(r"wiki/", include("wiki.urls")),
    ...
]
`
When I disable checking slug it works very well.. I can reach every page without any problem.

**Comment by benjaoming** [comment]
Let me rephrase:

> What do you see when you navigate to the slug that you are typing in?

What happens when you navigate to the slug that is rejected with "This slug conflicts with an existing URL"? What do you see?

**Comment by rokondo** [comment]
a) when I enter the url for the new article I saved and I navigate from the main page at `/wiki` everything works fine
b) when I enter a fantasy name after `/wiki/fantasy-xyz` then redirects to `/wiki/_create/?slug=fantasy-xyz`

Does it make a problem when I change the base url after installing and setting SITE_ID? I guess/assume that you use reversy url function of django to check the slug, so there should be no influence?

**Comment by benjaoming** [comment]
Both a) and b) sound like normal behavior.

Yes, django-wiki uses Django's `reverse` to check if a URL slug is "occupied" by some other application view.
